html/changelogs/.all_changelog.yml

```diff
@@ -7852,11 +7852,11 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - tweak: Hostile mobs can now determine if they can shoot through windows. They
       will check trajectory and if able to shoot through windows - will shoot.
   WowzewoW (credits to mattroks101 for original dropsound code, lighter sounds, Valve Software and Freesound.com.):
-  - soundadd: adds new emote sounds ( a lot )
```

Comment:
This change is ill-advised. Or you'd have to change the changelogs.html file as well.

Reply:
Ah dear